### ISSUE TITLE
[Feature request] convert object into topic string based on pattern

### ISSUE THREAD

**Issue by DrSensor** [RangerMauve/mqtt-pattern#2]
I think it will also be nice to have a function that does the reverse.
For example

`reverse(pattern : String, params : Object) : String | null`
```javascript
const pattern = 'device/+id/#data'
const params = {
	id: 'fitbit',
	data: ['rate', 'bmp']
}
var topic = MQTTPattern.reverse(pattern, params)

/* the `topic` will be
device/fitbit/rate/bpm
*/
```

**Comment by RangerMauve** [comment]
That's an awesome idea! I'll work on it as soon as I have time. :D In the meantime, feel free to beat me to it with a PR!

**Comment by RangerMauve** [comment]
Just added the new feature. Let me know what you think of it. (released in 1.2.0)
